packages/replicache-perf/package.json

```diff
@@ -17,7 +17,7 @@
     "@types/command-line-usage": "^5.0.2",
     "command-line-args": "^6.0.1",
     "command-line-usage": "^7.0.3",
-    "esbuild": "^0.24.2",
+    "esbuild": "^0.25.0",
```

Comment:
I thought we wanted to use `"esbuild": "0.25.0",`. In other words pin the version.